pkg/chartutil/create_test.go

```diff
@@ -26,6 +26,14 @@ import (
 	"k8s.io/helm/pkg/proto/hapi/chart"
 )
 
+func checkFile(t *testing.T, dir string, f string) {
+	if fi, err := os.Stat(filepath.Join(dir, f)); err != nil {
+		t.Errorf("Expected %s file: %s", f, err)
```

Comment:
With the other comment in place, this as well as line 33 can be changed to use `filepath.Base(f)` to get the file name.